Misc/NEWS.d/next/Windows/2019-07-01-12-38-48.bpo-10945.s0YBHG.rst

```diff
@@ -0,0 +1,2 @@
+Officially drop support for creating bdist_wininst installers on non-Windows
+systems.
```

Comment:
Technically, it wasn't supported previously, so I don't think that we even have to document the change. But I'm fine with keeping this changelog entry if you prefer.